src/pool/srv_pool.c

```diff
@@ -1535,6 +1535,7 @@ ds_pool_create_handler(crt_rpc_t *rpc)
 			D_GOTO(out_svc, rc);
 		}
 		svc->ps_rsvc.s_state = DS_RSVC_UP;
+		ABT_cond_broadcast(svc->ps_rsvc.s_state_cv);
```

Comment:
minor, seems can reuse the change_state() helper defined below

Reply:
Right. I chose not to export change_state to src/pool because we'll change src/pool to use the ds_rsvc_class.sc_bootstrap approach. Once that's done, s_state and s_state_cv will become internal to rsvc. The second paragraph of the commit message records this reason.